backend/inspirehep/hepdata/cli.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from datetime import datetime
+
+import click
+import requests
+from invenio_db import db
+
+from inspirehep.records.api import LiteratureRecord
+
+HEPDATA_URL = "https://www.hepdata.net/search/ids"
+
+
+@click.group()
+def hepdata():
+    """Commands for disambiguation"""
+
+
+@hepdata.command("harvest")
+@click.option(
+    "--since",
+    "-s",
+    "since",
+    default=None,
+    help="Date from when hepdata should be harvested. If not provided today's date would be used.",
```

Comment:
Maybe use yesterday instead? otherwise there's a gap between the beginning of the day and the harvest time. Another approach is to store the last run info like we do for CDS, but that might not be needed for such a simple thing.

Reply:
You need to update the docstring